site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -303,17 +297,17 @@ The length of time an SSO session can be inactive before it is closed.
 [field]#Logout URL# [optional]#Optional#::
 The URL the user is redirected to upon logout.
 
+[field]#Client credentials populate lambda# [optional]#Optional#::
```

Comment:
This was added when we added client credentials, so we should add a since annotation.